Changelog

```diff
@@ -1,3 +1,60 @@
+Unreleased - sledgehammer999 <sledgehammer999@qbittorrent.org> - v4.6.0
+    - FEATURE: Add (experimental) I2P support (glassez) cdded6cef7710fb25093ea142edea3d3b7479447
+    - FEATURE: Provide UI Theme editor (glassez) 77aa85fbd3af12b319f8f0b8372f4ba186d10c80
+    - FEATURE: Various UI theming improvements (glassez)
+    - FEATURE: Implement torrent tags editing dialog (glassez) d40be79c6933fd911644b0b0c833893ea480dc90
+    - FEATURE: Implement "Add torrent params" editing widget (glassez) 0cc29f18510b084b131fd0b02963a7aff909de01
+    - FEATURE: Revamp "Watched folder options" and "Automated RSS downloader" dialog (glassez) 93a1e58554fe9beff657b1302cc358722e8b0a1b 905f1416572913cf0bb0d143b7da381cd544fad8
+    - FEATURE: Allow to use another icons in dark mode (glassez) 72ac92ec682b96924bdec606bc4383430f3f7067
+    - FEATURE: Allow to add new torrents to queue top (glassez) 5d4766edbe88c768e8fa9a91ca3dadbf9a59aedf
+    - FEATURE: Allow to filter torrent list by save path (Tom) 0bb0829a9a22e4b468af6bcbc119df5d1d1571ec
+    - FEATURE: Expose 'socket send/receive buffer size' options (Chocobo1) 77411760a093bffafa3ae2c4f139990cd4ff55e6
+    - FEATURE: Expose 'max torrent file size' setting (Chocobo1) 66dfe8545dff47830fdc5c118c7e11d2cad21975
+    - FEATURE: Expose 'bdecode limits' settings (Chocobo1) 5a660fc8a979aa6b97c9059df4216da8b9a80a57
+    - FEATURE: Add options to adjust behavior of merging trackers to existing torrent (glassez) 4ef9a6444ae9067bb40fd2f5442564416e047689
+    - FEATURE: Add option to stop seeding when torrent has been inactive (Christopher) 35e18498d9bde2292f4c0be793f0870385e54770
+    - FEATURE: Allow to use proxy per subsystem (glassez) 6ac14d0c571995b3a3b750b9afdd6ff4c34797e9
+    - FEATURE: Allow to globally disable the use of proxy (glassez) 7ec80263e14a340395bf60f7da78c2cadd134b31
```

Comment:
If i'm not mistaken, this is the commit that returns the "(None)" proxy type. Non beta testers never knew it was gone.

Reply:
Generally no PRs marked as `Hotfix` should go into changelog.